File: src/targets/ruby/native/fixtures/nested.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har?foo[bar]=baz,zap&fiz=buz&key=value")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body

File: src/targets/go/native/fixtures/nested.go
package main

import (
	"fmt"
	"net/http"
	"io"
)

func main() {

	url := "http://mockbin.com/har?foo[bar]=baz,zap&fiz=buz&key=value"

	req, _ := http.NewRequest("GET", url, nil)

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := io.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}